linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/FunctionEmitter.scala

```diff
@@ -3287,6 +3718,9 @@ private[emitter] class FunctionEmitter(sjsGen: SJSGen) {
       }
     }
 
+    def isRTLongType(tpe: Type): Boolean =
```

Comment:
```suggestion
    def isSplitLongType(tpe: Type): Boolean =
```

I feel this reflects what this tests better: the type is such that we must split the long.